pages/servers.tsx

```diff
@@ -71,7 +117,7 @@ const ServerFilters = ({
 }) => {
   const intl = useIntl()
   const filterGroupMessages = defineMessages({
-    topic: { id: "server.filter_by.topic", defaultMessage: "Topic" },
+    category: { id: "server.filter_by.topic", defaultMessage: "Topic" },
```

Comment:
since `category` is used all over the api when we're constructing URLs and getting responses, i'm thinking we should keep that naming convention where we can